scripts/aem.js

```diff
@@ -441,11 +463,87 @@ function decorateIcons(element, prefix = '') {
   });
 }
 
+function openSelectedTab(tagsGroupSection, tabId, tabName) {
+  const tabContents = tagsGroupSection.querySelectorAll('.tab-content');
+  tabContents.forEach((tabContent) => {
+    tabContent.style.display = 'none';
+  });
+  const tabLinks = tagsGroupSection.querySelectorAll('.tab-link');
+  tabLinks.forEach((tabLink) => tabLink.classList.remove('active'));
+
+  const tabButton = tagsGroupSection.querySelector(`#${tabId}`);
+  tabButton.classList.add('active');
+  const selectedTabContents = tagsGroupSection.querySelectorAll(`[data-tab-name="${tabName}"]`);
+  selectedTabContents.forEach((tabContent) => {
+    tabContent.style.display = 'block';
+  });
+}
+
+function createTabButton(tabName, tagsGroupSection) {
+  const tabButtonId = tabName.toLowerCase().replace(/\s/g, '-');
+  const tabButton = createElement('button', { id: tabButtonId, class: 'tab-link' });
```

Comment:
this could be shortened further, with the text being the third parameter to createElement and onClick being added to the other element properties.